transferbot.sh

```diff
@@ -3,6 +3,17 @@
 set -eu
 set -o pipefail
 
+trap finish EXIT INT TERM
+
+finish () {
+  exit_code=$?;
+  if [ ! -z "${CALLBACK_ON_SUCCESS:-}" ] && [ "$exit_code" = 0 ]; then
```

Comment:
I'm a bit torn on whether this should be a single `CALLBACK_COMMAND` that gets passed the exit code as an argument, but considering this gets passed as an env var, it's probably a bit annoying to start adding conditions to that single line string...

Reply:
yeah, I can totally see that; probably easiest to do this